spec/support/request_spec_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module RequestSpecHelper
+  def json
```

Comment:
I would name this a bit different to give it a bit more context from the name. Maybe `response_json` or  `parsed_body` or anything else that would make it more obvious that the method refers to the response body parsed as JSON.